test/retrieve.jl

```diff
@@ -0,0 +1,27 @@
+datadir = joinpath(@__DIR__,"data")
+
+@testset "ERA5 data" begin
+    name = "reanalysis-era5-pressure-levels-monthly-means"
+    params = """{
+            'format': 'grib',
+            'product_type': 'monthly_averaged_ensemble_members_by_hour_of_day',
+            'variable': 'divergence',
+            'pressure_level': '775',
+            'year': '2020',
+            'month': '03',
+            'time': '12:00',
+        }"""
+    filepath = joinpath(datadir, "era5.grib")
+
+    @testset "Retrieve" begin
+        data = CDSAPI.retrieve(name, py2ju(params), filepath)
+        @test typeof(data) <: Dict
+        @test data["result_provided_by"] == "dd02ed70-03e3-4989-b5a3-9b551adbd4e3"
+        @test data["content_type"] == "application/x-grib"
+    end
+
+    @testset "Data" begin
```

Comment:
What do you think of reorganizing these testsets so that the top level test set is `Retrieve` and the subtestsets are the specific datasets like `ERA5`?

Reply:
okay, and then we would test it for multiple datasets, right?

Reply:
Exactly :+1:

Reply:
I'm splitting this up in two separate files, `retrieve.jl` will retrieve all datasets and `verify_data.jl`/`data.jl` will test the data.

Reply:
I would avoid making it too complex. To me a simple `retrieve.jl` with multiple `@testset` seems enough. Each testset will be small just checking the variables are present and other simple checks.

Reply:
sounds good.